manifests/nodeExporter-prometheusRule.yaml

```diff
@@ -247,6 +247,75 @@ spec:
       for: 15m
       labels:
         severity: critical
+    - alert: NodeCPUHighUsage
+      annotations:
+        description: |
+          CPU usage at {{ $labels.instance }} has been above 90% for the last 15 minutes, is currently at {{ printf "%.2f" $value }}%.
+        runbook_url: https://runbooks.prometheus-operator.dev/runbooks/node/nodecpuhighusage
+        summary: High CPU usage.
+      expr: |
+        sum without(mode) (avg without (cpu) (rate(node_cpu_seconds_total{job="node-exporter", mode!="idle"}[2m]))) * 100 > 90
+      for: 15m
+      labels:
+        severity: info
```

Comment:
Just briefly freaked out about seeing this alert, however, it's `severity: info` which shouldn't alert.